builtin/providers/aws/resource_aws_cloudwatch_log_group.go

```diff
@@ -49,10 +58,19 @@ func resourceAwsCloudWatchLogGroup() *schema.Resource {
 func resourceAwsCloudWatchLogGroupCreate(d *schema.ResourceData, meta interface{}) error {
 	conn := meta.(*AWSClient).cloudwatchlogsconn
 
-	log.Printf("[DEBUG] Creating CloudWatch Log Group: %s", d.Get("name").(string))
+	var logGroupName string
```

Comment:
I'm starting to think we should refactor this type of code into a common structure fund - it is repeated in a lot of places now - thoughts?

We could pass `name`, `name_prefix` and return the formatted value

Thoughts?